waspc/data/Generator/templates/sdk/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
Adding this file fixed one of the build errors from the description (the one with `import.meta.env`).